app/config.py

```diff
@@ -418,3 +418,5 @@ def sl_getenv(env_var: str, default_factory: Callable = None):
 ZENDESK_HOST = os.environ.get("ZENDESK_HOST")
 ZENDESK_API_TOKEN = os.environ.get("ZENDESK_API_TOKEN")
 ZENDESK_ENABLED = "ZENDESK_ENABLED" in os.environ
+
+ALLOWED_REDIRECT_DOMAINS = sl_getenv("ALLOWED_REDIRECT_DOMAINS", list) or []
```

Comment:
I wonder if `ALLOWED_REDIRECT_DOMAINS` isn't set, can we default its value to the hostname used in `URL` variable? This way the existing SL instance can benefit from this protection.

Reply:
Done :+1: